backend/experiment/static/block_admin.js

```diff
@@ -39,7 +42,34 @@ document.addEventListener("DOMContentLoaded", (event) => {
             { method: "POST", mode: 'same-origin', headers: { 'X-CSRFToken': csrftoken } })
 
         if (response.ok) {
-            location.reload()
+            location.reload();
         }
     }
 })
+
+/** Function to fix the headings for tabular inline forms
+  * @todo TODO: Remove this `fixHeadings` function once the issue with headings is fixed in `django-nested-admin`.
+  * - https://github.com/theatlantic/django-nested-admin/issues/261
+  * - https://github.com/theatlantic/django-nested-admin/pull/259
```

Comment:
It's now been merged!

Reply:
Ah yes! It wasn't mentioned anywhere on Github though 🙄